test/transaction.ts

```
@@ -1078,6 +1323,16 @@ describe('Transaction', () => {
       it('should inherit from Dml', () => {
         assert(transaction instanceof Dml);
       });
+
+      it('should throw am error when directedReadOptions are set', () => {
```

typo: should throw an error